doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst

```diff
@@ -121,7 +125,8 @@ the module too.
         module='numpy', docs_from_dispatcher=False, verify=False)
 
     @array_function_nodocs_from_c_func_and_dispatcher(_multiarray_umath.array)
-    def array(a, dtype=None, copy=True, order='K', subok=False, ndmin=0, like=None):
+    def array(a, dtype=None, copy=True, order='K', subok=False, ndmin=0, *,
```

Comment:
```suggestion
    def array(a, dtype=None, *, copy=True, order='K', subok=False, ndmin=0,
```
Minor nitpick: all arguments after `dtype` are keyword-only; not just `like`. https://github.com/numpy/numpy/blob/6bed9a91c1be207d2b69f13355449347d22c850b/numpy/core/_add_newdocs.py#L789

Reply:
Adjusted this now.